.github/workflows/ci-gradle.yml

```diff
@@ -7,20 +7,37 @@ on:
   release:
     types: [ released ]
 
-concurrency:
```

Comment:
This is used to cancel CI jobs for PRs that are updated at a later date with additional commits etc. Is there an alternate workflow we should follow for canceling these with Depot?